doc/man1/openssl-cmp.pod.in

```diff
@@ -1051,7 +1072,7 @@ So far this has no effect because the server does not accept any error messages.
 
 =item B<-accept_raverified>
 
-Accept RAVERIFED as proof-of-possession (POPO).
+Accept RAVERIFED as proof of possession (POPO).
```

Comment:
Removal of hyphens is inconsistent with other uses.

Reply:
My rationale for removing the hyphenation on "proof-of-possession" when it is used as a noun is that,
AFAIK, in English, compound nouns are not hyphenated but compound adjectives are.

Reply:
English tends to hyphenate compound nouns when they are used as adjectives, but not when used as a noun.

i.e. not being hyphenated would be better here

(For example, google "proof of work" vs "proof-of--work" - the former is more prevalent)

Reply:
Yeah, I was mostly noting the inconsistency. But I agree with @tom-cosgrove-arm after re-reading it a half-dozen times.